### ISSUE TITLE
Unmount in windows does not work

### ISSUE THREAD

**Issue by kapitainsky** [kapitainsky/RcloneBrowser#13]

**Comment by kapitainsky** [comment]
No sure though if this is browser issue or rclone itself. Would be nice if some windows users test

**Comment by kapitainsky** [comment]
Windows.... there is no easy way to do umount from within the browser. If somebody has ideas i would appreciate

**Comment by danoe** [comment]
@kapitainsky, glad you took over the project. I'll test it on my machine, though i have never tried the mount before. 

First I ran into missing winfsb, maybe a little popup with where to download, and if any special options are needed would be nice there.
```
panic: cgofuse: cannot find winfsp

goroutine 25 [running]:
github.com/rclone/rclone/vendor/github.com/billziss-gh/cgofuse/fuse.(*FileSystemHost).Mount(0xc0001104e0, 0xc00002a118, 0x2, 0xc000548000, 0x11, 0x14, 0x68200000200)
C:/gopath/src/github.com/rclone/rclone/vendor/github.com/billziss-gh/cgofuse/fuse/host.go:1153 +0x4e2
github.com/rclone/rclone/cmd/cmount.mount.func1(0xc0001104e0, 0xc00002a118, 0x2, 0xc000548000, 0x11, 0x14, 0x190eec0, 0xc0004f4a80, 0xc000066060)
C:/gopath/src/github.com/rclone/rclone/cmd/cmount/mount.go:153 +0x74
created by github.com/rclone/rclone/cmd/cmount.mount
C:/gopath/src/github.com/rclone/rclone/cmd/cmount/mount.go:151 +0x3aa
```

So I installed WinFsp, mounted the gdrive, copied down a couple files. Stopped the job to unmount, Rclone browser hung unresponsive for around 20 seconds. Then it came back, the job was gone, but the drive still mounted. (No other jobs running)

I closed rclone browser, it prompted there was a job running, if i'd like to stop it and end rclone browser, I pressed Ok and rclone browser closed and the mount disappeared.

```
Windows 10 Insider Preview (Build 18945) (Released 2019/07/26)
```

**Comment by kapitainsky** [comment]
You are right. Unmounting drives on Windows does not work at the moment. There is discussion on rclone [forum](https://forum.rclone.org/t/rclone-mount-progrmatic-umount-under-windows/12067) how to achieve it - there is no clean way to do this atm. In the meantime if you really have to mount a drive just run it from cmd.

**Comment by caspertone2003** [comment]
I have already installed the latest rclone beta with functionality included.
Ready to beta test if you wish the new rclonebrower functionality ;-)
CT

**Comment by kapitainsky** [comment]
Thank you! Still not ready - but next few days I should nail it at last.

**Comment by kapitainsky** [comment]
Actually I feel that it is ready - if you would like to try you can compile kptsky_testing branch.

**Comment by caspertone2003** [comment]
... i am unable to compile anything...
CT

**Comment by kapitainsky** [comment]
ok. when have some time I will think how to streamline beta releases…

**Comment by caspertone2003** [comment]
you can just produce a quick and dirty zip and put it here... then you can delete the comment/remove the zip later.

**Comment by kapitainsky** [comment]
It is not critical, There are three operating systems… i know this bug is about windows but I have already found out that often things fixed for one OS break things for other. And Windows is the worst  - with very weird limitations and dependencies.

So if somebody wants to test early version must be prepared to compile from source. Simply I have no time at this stage.

**Comment by caspertone2003** [comment]
Understood...

**Comment by kapitainsky** [comment]
Fixed in just released [1.6.0](https://github.com/kapitainsky/RcloneBrowser/releases/tag/1.6.0)

**Comment by caspertone2003** [comment]
Thanks, good achievement!
CT
